cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/internal/replication/PullStrategy.java

```diff
@@ -273,12 +268,21 @@ private void replicate()
 
         long endTime = System.currentTimeMillis();
         long deltaTime = endTime - startTime;
-        String msg = String.format(
-                "Pull completed in %sms (%s total changes processed)",
-                deltaTime,
-                this.state.documentCounter
-        );
+        String msg;
+        if (this.state.cancel) {
+            msg = String.format(Locale.ENGLISH,
+                    "Pull canceled after %sms (%s changes processed",
```

Comment:
Missing right paren and this message differs from the other cancellation message